sound/soc/sof/ipc4-ops.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
```

Comment:
why me?

Reply:
I will change it to me. The "Author" tag seems to be not necessary. Can we just remove it?

Reply:
I always use tig blame...

Reply:
Yes, not needed